sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -4,6 +4,20 @@ var srlog = SIREPO.srlog;
 var srdbg = SIREPO.srdbg;
 
 SIREPO.NUMBER_REGEXP = /^\s*(\-|\+)?(\d+|(\d*(\.\d*)))([eE][+-]?\d+)?\s*$/;
+SIREPO.SAFE_PATH_REGEXP_EXCLUDE_CHARS = [   // array of illegal filename characters
+    '\\\\',  // doubly escaped
```

Comment:
I'll take care of all this - I was pondering as I went whether they really need to be globals, since they are likely only to be used in one place, but the existing number regex was there so...